eng/Versions.props

```diff
@@ -10,7 +10,7 @@
     <MicrosoftDotNetBuildTasksTemplatingPackageVersion>10.0.0-beta.24604.4</MicrosoftDotNetBuildTasksTemplatingPackageVersion>
     <MicrosoftTestingExtensionsCodeCoverageVersion>17.13.1</MicrosoftTestingExtensionsCodeCoverageVersion>
     <!-- comment to facilitate merge conflicts -->
-    <MicrosoftTestingExtensionsRetryVersion>1.5.0-preview.24626.1</MicrosoftTestingExtensionsRetryVersion>
+    <MicrosoftTestingExtensionsRetryVersion>1.5.0-preview.24627.3</MicrosoftTestingExtensionsRetryVersion>
```

Comment:
@Evangelink This is 3.7 branch. Is it intended this is a preview version? I thought this should have been `1.5.0`?

Reply:
It was 1.5.0, but as we prepare the bug fix, we need to let TA <-> MSTest insertion flow